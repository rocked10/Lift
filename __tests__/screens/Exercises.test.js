import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Exercises from '../../screens/exercises';

describe('Testing Exercises Screen', () => {
    const wrapper = renderer.create(<Exercises />);

    it('Should render', () => {
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});