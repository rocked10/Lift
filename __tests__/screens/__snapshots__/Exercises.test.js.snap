// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Exercises Screen Should render 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 16,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderRadius": 4,
        "elevation": 4,
        "flexDirection": "row",
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 3,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 4,
      }
    }
  >
    <View
      accessibilityLabel="search"
      accessibilityRole="button"
      accessibilityState={
        Object {
          "disabled": undefined,
        }
      }
      accessible={true}
      focusable={true}
      hitSlop={
        Object {
          "bottom": 6,
          "left": 6,
          "right": 6,
          "top": 6,
        }
      }
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "margin": 6,
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 18,
              "height": 36,
              "width": 36,
            },
            undefined,
            undefined,
          ],
        ]
      }
    >
      <View>
        <Text />
      </View>
    </View>
    <TextInput
      accessibilityRole="search"
      accessibilityTraits="search"
      allowFontScaling={true}
      keyboardAppearance="light"
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Search for an exercise..."
      placeholderTextColor="rgba(0, 0, 0, 0.54)"
      rejectResponderTermination={true}
      returnKeyType="search"
      selectionColor="#6200ee"
      style={
        Array [
          Object {
            "alignSelf": "stretch",
            "flex": 1,
            "fontSize": 18,
            "minWidth": 0,
            "paddingLeft": 8,
            "textAlign": "left",
          },
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "fontFamily": "lato-regular",
          },
        ]
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessibilityLabel="clear"
      accessibilityRole="button"
      accessibilityState={
        Object {
          "disabled": true,
        }
      }
      accessible={true}
      focusable={true}
      hitSlop={
        Object {
          "bottom": 6,
          "left": 6,
          "right": 6,
          "top": 6,
        }
      }
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "margin": 6,
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 18,
              "height": 36,
              "width": 36,
            },
            Object {
              "opacity": 0.32,
            },
            undefined,
          ],
        ]
      }
    >
      <View>
        <Text />
      </View>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "flex-end",
        "marginTop": 12,
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
  <RCTScrollView
    showsVerticalScrollIndicator={false}
  >
    <View>
      <View
        style={
          Array [
            Object {
              "marginVertical": 8,
            },
            undefined,
          ]
        }
        theme={
          Object {
            "animation": Object {
              "scale": 1,
            },
            "colors": Object {
              "accent": "#03dac4",
              "backdrop": "rgba(0, 0, 0, 0.5)",
              "background": "#f6f6f6",
              "disabled": "rgba(0, 0, 0, 0.26)",
              "error": "#B00020",
              "notification": "#f50057",
              "onSurface": "#000000",
              "placeholder": "rgba(0, 0, 0, 0.54)",
              "primary": "#6200ee",
              "surface": "#ffffff",
              "text": "#000000",
            },
            "dark": false,
            "fonts": Object {
              "light": Object {
                "fontFamily": "System",
                "fontWeight": "300",
              },
              "medium": Object {
                "fontFamily": "System",
                "fontWeight": "500",
              },
              "regular": Object {
                "fontFamily": "System",
                "fontWeight": "400",
              },
              "thin": Object {
                "fontFamily": "System",
                "fontWeight": "100",
              },
            },
            "roundness": 4,
          }
        }
      >
        <Text
          numberOfLines={1}
          style={
            Array [
              Object {
                "color": "#000000",
                "fontFamily": "System",
                "fontWeight": "400",
              },
              Object {
                "textAlign": "left",
              },
              Array [
                Object {
                  "paddingHorizontal": 16,
                  "paddingVertical": 13,
                },
                Object {
                  "color": "rgba(0, 0, 0, 0.54)",
                  "fontFamily": "System",
                  "fontWeight": "500",
                },
                Object {
                  "fontFamily": "lato-bold",
                  "marginTop": -6,
                },
              ],
            ]
          }
        >
          Exercises
        </Text>
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#f6f6f6",
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "padding": 8,
                    },
                    undefined,
                  ],
                ]
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "margin": 8,
                        "width": 40,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      Object {
                        "flex": 1,
                        "justifyContent": "center",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontFamily": "System",
                          "fontWeight": "400",
                        },
                        Object {
                          "textAlign": "left",
                        },
                        Array [
                          Object {
                            "fontSize": 16,
                          },
                          Object {
                            "color": "rgba(0, 0, 0, 0.87)",
                          },
                          Object {
                            "fontFamily": "lato-bold",
                          },
                        ],
                      ]
                    }
                  >
                    Olympic
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#f6f6f6",
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "padding": 8,
                    },
                    undefined,
                  ],
                ]
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "margin": 8,
                        "width": 40,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      Object {
                        "flex": 1,
                        "justifyContent": "center",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontFamily": "System",
                          "fontWeight": "400",
                        },
                        Object {
                          "textAlign": "left",
                        },
                        Array [
                          Object {
                            "fontSize": 16,
                          },
                          Object {
                            "color": "rgba(0, 0, 0, 0.87)",
                          },
                          Object {
                            "fontFamily": "lato-bold",
                          },
                        ],
                      ]
                    }
                  >
                    Legs
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#f6f6f6",
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "padding": 8,
                    },
                    undefined,
                  ],
                ]
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "margin": 8,
                        "width": 40,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      Object {
                        "flex": 1,
                        "justifyContent": "center",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontFamily": "System",
                          "fontWeight": "400",
                        },
                        Object {
                          "textAlign": "left",
                        },
                        Array [
                          Object {
                            "fontSize": 16,
                          },
                          Object {
                            "color": "rgba(0, 0, 0, 0.87)",
                          },
                          Object {
                            "fontFamily": "lato-bold",
                          },
                        ],
                      ]
                    }
                  >
                    Chest
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#f6f6f6",
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "padding": 8,
                    },
                    undefined,
                  ],
                ]
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "margin": 8,
                        "width": 40,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      Object {
                        "flex": 1,
                        "justifyContent": "center",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontFamily": "System",
                          "fontWeight": "400",
                        },
                        Object {
                          "textAlign": "left",
                        },
                        Array [
                          Object {
                            "fontSize": 16,
                          },
                          Object {
                            "color": "rgba(0, 0, 0, 0.87)",
                          },
                          Object {
                            "fontFamily": "lato-bold",
                          },
                        ],
                      ]
                    }
                  >
                    Back
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#f6f6f6",
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "padding": 8,
                    },
                    undefined,
                  ],
                ]
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "margin": 8,
                        "width": 40,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      Object {
                        "flex": 1,
                        "justifyContent": "center",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontFamily": "System",
                          "fontWeight": "400",
                        },
                        Object {
                          "textAlign": "left",
                        },
                        Array [
                          Object {
                            "fontSize": 16,
                          },
                          Object {
                            "color": "rgba(0, 0, 0, 0.87)",
                          },
                          Object {
                            "fontFamily": "lato-bold",
                          },
                        ],
                      ]
                    }
                  >
                    Arms
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#f6f6f6",
              }
            }
          >
            <View
              accessibilityRole="button"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "overflow": "hidden",
                  },
                  Array [
                    Object {
                      "padding": 8,
                    },
                    undefined,
                  ],
                ]
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "margin": 8,
                        "width": 40,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      Object {
                        "flex": 1,
                        "justifyContent": "center",
                      },
                    ]
                  }
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      Array [
                        Object {
                          "color": "#000000",
                          "fontFamily": "System",
                          "fontWeight": "400",
                        },
                        Object {
                          "textAlign": "left",
                        },
                        Array [
                          Object {
                            "fontSize": 16,
                          },
                          Object {
                            "color": "rgba(0, 0, 0, 0.87)",
                          },
                          Object {
                            "fontFamily": "lato-bold",
                          },
                        ],
                      ]
                    }
                  >
                    Cardio
                  </Text>
                </View>
                <View
                  style={
                    Array [
                      Object {
                        "margin": 8,
                      },
                      undefined,
                    ]
                  }
                >
                  <Text />
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
